src/parameter_declarations.ts

```diff
@@ -52,7 +62,77 @@ Parameter.declare({
 
 Parameter.declare({
 	name: DriveAddressParameter,
-	aliases: ['-a', '-drive-address'],
+	aliases: ['-a', '--drive-address'],
 	description: 'the address',
 	forbiddenConjunctionParameters: [DrivePasswordParameter, DriveKeyParameter]
 });
+
+Parameter.declare({
+	name: DriveIdParameter,
+	aliases: ['-d', '--drive-id'],
+	description: 'the drive ID to get metadata from',
+	required: true
+});
+
+Parameter.declare({
+	name: ArAmountParameter,
+	aliases: ['-a', '--ar-amount'],
+	description: 'required: amount of AR to send',
+	required: true
+});
+
+Parameter.declare({
+	name: DestinationAddressParameter,
+	aliases: ['-d', '--dest-address'],
+	description: 'required: destination wallet address',
+	required: true
+});
+
+Parameter.declare({
+	name: TransactionIdParameter,
+	aliases: ['-t', '--tx-id'],
+	description: 'The transaction id to check the status of in the mempool',
+	required: true
+});
+
+Parameter.declare({
+	name: ConfirmationsParameter,
+	aliases: ['-c', '--confirmations'],
+	description: 'Number of confirmations to determine if a transaction is mined'
+});
+
+Parameter.declare({
+	name: ParentFolderIdParameter,
+	aliases: ['--parent-folder-id'],
```

Comment:
This should have -f as a short option. The ones below are missing their short options from below as well: 

```ts
// old index.ts

.requiredOption(
	'-f, --parent-folder-id <parent folder id>',
	`the ArFS folder ID for the folder in which this file will reside (i.e. its parent folder)
	• To upload the file to the root of a drive, use the root folder ID of the drive`
)
.option(
	'-l, --local-file-path <local file path>',
	`the path on the local filesystem for the file that will be uploaded`
)
.option(
	'-d, --dest-file-name <destination file name>',
	`(OPTIONAL) a destination file name to use when uploaded to ArDrive`
)
```

Reply:
Yep! I missed some options. Added at 96584cc700f98fa16c9b26cb898cd5297063ac33